starklings-backend/config.py

```diff
@@ -0,0 +1,33 @@
+from os import environ, path
+from dotenv import load_dotenv
+
+basedir = path.abspath(path.dirname(__file__))
+load_dotenv(path.join(basedir, '.env'))
+
+class Config:
+    """Base config."""
+    SECRET_KEY = environ.get('SECRET_KEY')
+    host = environ.get('DATABASE_HOST', '')
+    database = environ.get('DATABASE_NAME', '')
+    user = environ.get('DATABASE_USER', '')
+    password = environ.get('DATABASE_PWD', '')
+    SQLALCHEMY_DATABASE_URI = f"mysql+pymysql://{user}:{password}@{host}/{database}"
```

Comment:
Maybe you could directly have a `DATABASE_URL` env var?

Reply:
I thought it more in a best practice way, depends on how we want to handle it, I can do that if needed !

Reply:
No prob, I'm just used to see the whole URL as an env var, but both ways are acceptable